libraries/client/client_webhook.py

```diff
@@ -187,6 +192,12 @@ def process_webhook(self):
             source_url = self.build_multipart_source(file_key, book)
             identifier, job = self.send_job_request_to_tx_manager(commit_id, source_url, rc, repo_name, repo_owner,
                                                                   count=book_count, part=i, book=book)
+
+            # Send lint request to tx-manager
+            lint_results = self.send_lint_request_to_tx_manager(job, source_url)
```

Comment:
So question for you: Can we call the linter on each individual book, one at a time, separate lambda calls? Or do we need to lint the whole repo? I wasn't going to send the DCS repo zip to USFM (Bible) and OBS linters, since often they are chunked (tS repos), so I am sending them the massaged zip file in the tx-client-webhook bucket.